webui/react/src/components/UPlot/UPlotChart.module.scss

```diff
@@ -0,0 +1,5 @@
+.base.dark {
+  :global(.u-select) {
+    background-color: rgba(255, 255, 255, 0.1);
```

Comment:
good fine

Reply:
🙏 I asked in uPlot issues, and they answered right away. Super responsive (unlike some parallel coordinates person)